plugin.video.mediathekview/resources/lib/kodi/kodiaddon.py

```diff
@@ -36,10 +37,16 @@ def __init__(self):
         self.icon = self.addon.getAddonInfo('icon')
         self.fanart = self.addon.getAddonInfo('fanart')
         self.version = self.addon.getAddonInfo('version')
-        self.path = mvutils.py2_decode(self.addon.getAddonInfo('path')) ##TODO self.unicodePath = unicode(self.path, 'utf-8')
-        self.datapath = mvutils.py2_decode(xbmc.translatePath(self.addon.getAddonInfo('profile'))) ### TODO.decode('utf-8')
+        self.path = mvutils.py2_decode(self.addon.getAddonInfo('path'))
+        ## TODO fix me
```

Comment:
You can check python 3 or matrix version jere

Reply:
done